src/quick-lint-js/translation-table-generated.h

```diff
@@ -1,11 +1,12 @@
+// Code generated by tools/compile-translations.go. DO NOT EDIT.
+// source po/*.po
```

Comment:
Shouldn't this have a colon:
```suggestion
// source: po/*.po
```